bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -37,7 +37,7 @@
 
 # We want iPython to take over the graphics
 ROOT.gROOT.SetBatch()
-
+ROOT.gROOT.SetWebDisplay("jupyter")
```

Comment:
What is this setting used for? I guess it does not have any effect in the old graphics.

Reply:
This is needed for `RWebWindow` - otherwise it will create `THttpServer` and will try to popup another web browser